### ISSUE TITLE
Improve important notice section

### ISSUE THREAD

**Issue by Abhey** [ContriHUB/Sac-Portal#1]
There is an important notice section on the landing page of the SAC portal, try to improve it. Currently, this moves only upwards, while it should be kind of circular.
[Sac MNNIT](http://sac.mnnit.ac.in/)

**Comment by ghost** [comment]
I'd like this issue assigned to me, Sir.

**Comment by hiiiideepak** [comment]
@AzDan yeah sure! you have to fix it such that it does a cyclic rotation. and also make notices a link. font size etc.

**Comment by AzDan** [comment]
Sir, I think I have the issue fixed, but I am unable to deploy it on Heroku. It says that I am using an older version of Python. What will be a viable solution for this?

**Comment by roopbiswas** [comment]
Sir , is this issue still open? If it is , please assign me this issue , I would love to work on it.
